etc/docs/template/layouts/404.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {{ partial "meta.html"}}
+    <link rel="shortcut icon" href="{{.Site.BaseURL}}/img/favicon.png">
+
+    <title>{{.Title}}</title>
+
+    {{ partial "assets/css.html" . }}
+  </head>
+
+  <body>
+  <!--header start-->
+  {{ partial "header/main.html" . }}
+  <!--header end-->
+
+  <div id="promo" class="jumbotron">
+    <div class="container">
+    <div class="row">
+      <div class="col-md-2"></div>
+      <div class="col-md-6">
+        <h1>404: Page not found</h1>
```

Comment:
Just a quick win idea, a link back to the homepage or mongodb manual could be good here, if we end up having to break legacy version links down the road this page may get an uptick in visits.

Reply:
Do you mind if I make this change in my follow up PR?  There's no way to test that the link I add works with the changes in this PR (since we don't actually have the whole site included).